README.md

```diff
@@ -13,23 +13,10 @@
   - [🛠 Built With ](#-built-with-)
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own created logo at the top of your readme file to present yourself not just as a software developer but also as a brand in the tech industry 👍